website/docs/config/networks.mdx

```diff
@@ -3,7 +3,7 @@ title: Testnet and Mainnet Configuration
 ---
 
 :::note
-This document details the configuration and use of Tezos testnets and mainnet. If you are looking for information on running or originating to a sandbox, please see the Flextesa documentation [here](docs/plugins/plugin-flextesa)
+This document details the configuration and use of Tezos testnets and mainnet. If you are looking for information on running or originating to a sandbox, please see [the Flextesa plugin documentation](https://taqueria.io/docs/plugins/plugin-flextesa/).
```

Comment:
With this change, if someone is checking out the preview build of the website, they will be redirected to the main site. They might get confused. Does a root-relative path work for here?

Reply:
It doesn't.